localization/english/wc_nicknames_l_english.yml

```
@@ -1,4 +1,86 @@
 ﻿l_english:
  nick_the_prophet:0 "the Prophet"
  nick_the_harbinger_of_the_void:0 "the Harbinger of the Void"
- nick_the_whacked:0 "the Whacked"
\ No newline at end of file
+ nick_the_ashbringer:0 "the Ashbringer"
+ nick_the_thunderbringer:0 "the Thunderbringer"
+ nick_the_adamant:0 "the Adamant"
+ nick_the_australian:0 "the Australian"
+ nick_the_whacked:0 "the Whacked"
+ nick_the_lightbringer:0 "the Lightbringer"
+ nick_the_dire:0 "the Dire"
+ nick_the_spiritbinder:0 "the Spiritbinder"
+ nick_the_old_murkeye:0 "the Old Murkeye"
+ nick_the_defiler:0 "the Defiler"
+ nick_the_deceiver:0 "the Deceiver"
+ nick_the_darkener:0 "the Darkener"
+ nick_the_ice_hearted:0 "the Ice-Hearted"
```

As above, I'd suggest changing this to Cold Hearted as it is a more common phrase in English than Ice Hearted.
```suggestion
 nick_the_cold_hearted:0 "the Cold-Hearted"
```

---

Basically it's a historical name
https://wow.gamepedia.com/Halfdan_the_Ice-Hearted

---

Ahh, fair enough. Thats fine as is then. My bad.